tidb-cloud/integrate-tidbcloud-with-cloudflare.md

```diff
@@ -1,192 +1,102 @@
 ---
 title: Integrate TiDB Cloud with Cloudflare
-summary: Learn how deploy Cloudflare Workers with TiDB Cloud.
+summary: Learn how to deploy Cloudflare Workers with TiDB Cloud.
 ---
 
 # Integrate TiDB Cloud with Cloudflare Workers
 
 [Cloudflare Workers](https://workers.cloudflare.com/) is a platform that allows you to run code in response to specific events, such as HTTP requests or changes to a database. Cloudflare Workers is easy to use and can be used to build a variety of applications, including custom APIs, serverless functions, and microservices. It is particularly useful for applications that require low-latency performance or need to scale quickly.
 
-However, you may find it hard to connect to TiDB Cloud from Cloudflare Workers because Cloudflare Workers runs on the V8 engine which cannot make direct TCP connections.
+You may find it hard to connect to TiDB Cloud from Cloudflare Workers because Cloudflare Workers runs on the V8 engine which cannot make direct TCP connections. You can use [TiDB Cloud serverless driver](/tidb-cloud/serverless-driver.md) helps you connect to Cloudflare Workers over HTTP connection.
```

Comment:
```suggestion
You may find it hard to connect to TiDB Cloud from Cloudflare Workers because Cloudflare Workers runs on the V8 engine which cannot make direct TCP connections. You can use [TiDB Cloud serverless driver](/tidb-cloud/serverless-driver.md) to help you connect to Cloudflare Workers over HTTP connection.
```